### ISSUE TITLE
Array types cause breakage in asciidoc output

### ISSUE THREAD

**Issue by mrice32** [OpenZeppelin/solidity-docgen#71]
Array types leave an unescaped close bracket in the asciidoc output - see the OZ documentation for an example of [this breakage](https://docs.openzeppelin.com/contracts/2.x/api/token/erc777#IERC777-defaultOperators--). Not sure if you have a preferred solution, but one way to fix it is to add a handlebars helper to escape the close bracket and allow template authors to call it whenever outputting solidity types. You could add something like this to `handlebars.ts`:
```
    escapeCloseBracket: function (input) {
        return input.replace("]", "\\]");
    }
```

Happy to open a PR if you think this solution makes sense.

**Comment by mrice32** [comment]
Yup, we're using asciidoc for our site! We've taken advantage of a lot of your tooling - it's been super helpful.

Totally agree that a deeper solution that sanitizes the types would be preferable. We implemented the solution I presented above in our fork as a quick workaround to avoid having to dig deeper into the codebase.

**Comment by frangio** [comment]
I've left a comment in your repository in https://github.com/UMAprotocol/protocol/pull/1046#issuecomment-601902777. Reproducing it here:

I've realized that a simple change in the template fixes it for us. I've removed the `pass:normal` macro in the places where it wasn't necessary. See [OpenZeppelin/openzeppelin-contracts#2141](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/2141/files).

If more escaping issues show up elsewhere I will reconsider providing a helper.
